### ISSUE TITLE
Containerising an existing website using this implementation gives installation errors

### ISSUE THREAD

**Issue by cloudgrimm** [stcox/charts#15]
I am trying to recreate the exact steps you have done with a fresh wordpress installation to an existing wordpress site residing in a VM. The following are the steps I have taken:

- Extract codebase using ftp from the VM
-ignore the uploads folder with media content stored on GCS
-Zip the file then upload to GCS and make it public.
-substitute the wordpress download from wordpress.org to my GCS bucket.
-create customised images using your dockerfile implementations
-Follow the k8s implementation.

The images are build successfully without any trouble. 

Please check the following error logs, maybe I am missing something because I continuously get a crashloopback error. 
 
<img width="956" alt="screenshot 2019-01-28 at 23 26 09" src="https://user-images.githubusercontent.com/22336209/51867492-5910a980-2354-11e9-80c7-5c89b385d4e2.png">

**Comment by stcox** [comment]
To migrate a site to GKE WordPress, you should:

1. Create a new site in GKE WordPress, dropping the existing (empty) db and importing the old (migration) db.

2. Use `kubectl cp` to copy the migrating wp-content directory up to the appropriate kubernetes pod. 

3. Use the [WP-Stateless plugin](https://wordpress.org/plugins/wp-stateless/) to serve your uploads from GCS.

I don't think it's possible to run WordPress core from GCS, because it requires a different file system.

**Comment by cloudgrimm** [comment]
Good day,

I think you misunderstood me when I said I was serving the site in GCS.

So this is the current setup:

1. Media upload content is being served in GCS

2. Database is now being served by cloudSql

3. The new site I packaged the whole drupal core files of the new site
using a new image as a whole. So I just zipped the new site with all the
configs and other files as I would have had if it was a
Fresh WP site from WordPress.org.

Then like how you used to create the first image without the redis
extensions that is how I created my first word press image with the new
code base. Image is created successfully

After that I created the second image with the redis extensions as you did
it too. Image is created successfully

Now moving everything to K8s then causes the error logs I highlighted
earlier.

But this approach might be tedious and error prone.

This
Tinashe Wilbrod Chipomho
